pages/about.js

```diff
@@ -0,0 +1,112 @@
+//global imports
+import React from "react";
+import Layout from "../src/components/layouts/aboutLayout";
+import { Typography, Container } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+//data
+import info from "../public/data/info.json";
+import members from "../public/data/members.json";
+import links from "../public/data/links.json";
+//local components
+import AboutHeaderImage from "../src/components/about/AboutHeaderImage";
+import InfoBubble from "../src/components/about/InfoBubble";
+import InfoLink from "../src/components/about/InfoLink";
+import Member from "../src/components/about/Member";
+
+const backgroundImage = `url("images/about_background.png")`;
+const useStyles = makeStyles(theme => {
+  return {
+    root: {
```

Comment:
`root` implies that it's on the root element, but we're applying this in several places. Can we rename this to something like `centeredText`?